### ISSUE TITLE
When "My Order" column is too long, it becomes impossible to scroll to the bottom

### ISSUE THREAD

**Issue by unfulvio** [woothemes/storefront#398]
### tldr

when I have too many items in thie "My Order" checkout page column, this section may become so long that it's unscrollable once I reach the bottom of the page (see screenshot below)

### What happened:

I was testing out something for Checkout Add Ons WooCommerce Extension, which adds fields to an order at checkout. When a customer selects some of them, these are also put into the checkout summary under "My Order" in Checkout page. 

![screenshot from 2016-05-19 12-54-42](https://cloud.githubusercontent.com/assets/1227930/15396781/e5844108-1ddd-11e6-9a22-6d6615d16ff3.png)

I'm unable to reach the payment options or the purchase button, posing a UX issue

#### To replicate:

Have many payment options, and try some extension that add up items in the summary, such as Checkout Add Ons. On Checkout page, have a customer select or enter values in some add ons and try to scroll down the page. 

---

I hope this is not a duplicate or a known issue, I have tried to look around but wasn't sure how to search for it. 

I found this while on latest Google Chrome for Linux

**Comment by BurlesonBrad** [comment]
> I hope this is not a duplicate or a known issue...

....well.... #352 & #347 :cry: Apparently it **IS** both _known_ and (somewhat) _recurring_ as well

There's a growing part of me that wants to totally roll back Storefront's Checkout files to
# NOTHING
# BUT
# SIMPLE
...this :hankey: is gettin' to be consistent. #JustSayin'

**Comment by BurlesonBrad** [comment]
@jameskoster, This demo http://demo.woothemes.com/storefront/checkout/ is ....well.... let's be honest: To be more realistic, it would / could / should have Stripe and PayPal fields (even if in Sandbox mode) enabled, and a few shipping selections to be a better demo.

Seeing recurring issues with products and extensions **outside** of the _WooThemes Umbrella_ is inevitable and expected.

Recurring issues with products and extensions **WITHIN WooTheme's own house** should be preventable and can be alleviated with a better workflow.

**How can I help this process?**

**Comment by jameskoster** [comment]
Hmm, probably safest to just disable this feature if the Checkout Add-ons extension is active. Will add this to the next release. For now you can disable like so; https://docs.woothemes.com/document/storefront-faq/#i-dont-like-the-sticky-order-review-feature-on-checkout-can-i-remove-it

@BurlesonBrad you're already helping a great deal just by opening issues here on github :)
